core/src/impl/Cabana_PerformanceTraits.hpp

```diff
@@ -83,6 +83,16 @@ class PerformanceTraits<Kokkos::Experimental::HIP>
 };
 #endif
 
+//---------------------------------------------------------------------------//
+#if defined( KOKKOS_ENABLE_OPENMPTARGET )
+template <>
+class PerformanceTraits<Kokkos::Experimental::OpenMPTarget>
+{
+  public:
+    static constexpr int vector_length = 16;
```

Comment:
I would add a FIXME because this will depend on the architecture we are targeted.

Reply:
Done